client/src/test/test.kao

```diff
@@ -117,7 +131,7 @@ Person CEDRIC_V
 				--> referential {	"Cédric V.",
 									"Cedrick V."
 								 }
-				--> anaphor		PERSONAL_PRONOUN
+				--> anaphor		PERSONAL_PRONOUN.getCityName()
```

Comment:
What's that? 8-D

Reply:
In this file, I just write things that are OK syntactically speaking, even if it won't compile because the symbols don't exist. 
If you want, I can still remove this change or add a complete that will make it possible :D

Reply:
I don't like the complete :-p But if you just add a comment, we are good to go :-)